Latest commit doesn't build

I was looking over all the issues for hacktoberfest but I wasn't able to get the latest commit to build. I am able to run the `build.ps1` script but when I run `dotnet run` I get the following errors:

```
Exercises\Clock.cs(26,21): error CS0200: Property or indexer 'CanonicalDataCase.ConstructorInput' cannot be assigned to -- it is read only [C:\Users\bever
ett\Documents\Github\csharp\generators\Generators.csproj]
Exercises\Clock.cs(34,21): error CS0200: Property or indexer 'CanonicalDataCase.ConstructorInput' cannot be assigned to -- it is read only [C:\Users\bever
ett\Documents\Github\csharp\generators\Generators.csproj]
Exercises\Clock.cs(35,45): error CS7036: There is no argument given that corresponds to the required formal parameter 'value' of 'CollectionExtensions.Rem
ove<TKey, TValue>(IDictionary<TKey, TValue>, TKey, out TValue)' [C:\Users\beverett\Documents\Github\csharp\generators\Generators.csproj]
Exercises\Clock.cs(38,21): error CS0200: Property or indexer 'IReadOnlyDictionary<string, dynamic>.this[string]' cannot be assigned to -- it is read only
[C:\Users\beverett\Documents\Github\csharp\generators\Generators.csproj]

The build failed. Please fix the build errors and run again.
```

I checked out the previous commit `git checkout d2bcc1facd` and ran `dotnet run` and it built just fine.

--- a/generators/Exercises/Clock.cs
+++ b/generators/Exercises/Clock.cs
@@ -1,61 +1,55 @@
 ﻿using System.Collections.Generic;
 using Generators.Input;
 using Generators.Output;
-using Newtonsoft.Json.Linq;
 
 namespace Generators.Exercises
 {
     public class Clock : Exercise
     {
-        private const string paramClock1 = "clock1";
-        private const string paramClock2 = "clock2";
-        private const string paramHour = "hour";
-        private const string paramMinute = "minute";
+        private const string ParamClock1 = "clock1";
+        private const string ParamClock2 = "clock2";
+        private const string ParamHour = "hour";
+        private const string ParamMinute = "minute";
 
-        private const string propertyCreate = "create";
-        private const string propertyEqual = "equal";
-        private const string propertyEquals = "equals";
-        private const string propertyToString = "to_string";
+        private const string PropertyCreate = "create";
+        private const string PropertyEqual = "equal";
+        private const string PropertyEquals = "equals";
+        private const string PropertyToString = "to_string";
 
         protected override void UpdateCanonicalData(CanonicalData canonicalData)
         {
             foreach (var canonicalDataCase in canonicalData.Cases)
             {
-                if (canonicalDataCase.Property != propertyEqual)
+                if (canonicalDataCase.Property != PropertyEqual)
                 {
-                    canonicalDataCase.ConstructorInput = new Dictionary<string, object>
-                    {
-                        [paramHour] = canonicalDataCase.Input[paramHour],
-                        [paramMinute] = canonicalDataCase.Input[paramMinute]
-                    };
+                    canonicalDataCase.SetConstructorInputParameters(ParamHour, ParamMinute);
                 }
                 else
                 {
-                    canonicalDataCase.ConstructorInput = ((JObject)canonicalDataCase.Input[paramClock1]).ToObject<Dictionary<string, object>>();
-                    canonicalDataCase.Input.Remove(paramClock1);
+                    canonicalDataCase.SetConstructorInputParameters(ParamClock1);
 
-                    var result = ((JObject)canonicalDataCase.Input[paramClock2]).ToObject<Dictionary<string, object>>();
-                    canonicalDataCase.Input[paramClock2] = new UnescapedValue($"new Clock({result[paramHour]}, {result[paramMinute]})");
+                    var result = (Dictionary<string, object>)canonicalDataCase.Properties[ParamClock2];
+                    canonicalDataCase.Properties[ParamClock2] = new UnescapedValue($"new Clock({result[ParamHour]}, {result[ParamMinute]})");
                 }
 
-                if (canonicalDataCase.Property == propertyCreate)
+                if (canonicalDataCase.Property == PropertyCreate)
                 {
-                    canonicalDataCase.Property = propertyToString;
+                    canonicalDataCase.Property = PropertyToString;
                 }
-                else if (canonicalDataCase.Property == propertyEqual)
+                else if (canonicalDataCase.Property == PropertyEqual)
                 {
-                    canonicalDataCase.Property = propertyEquals;
+                    canonicalDataCase.Property = PropertyEquals;
                 }
             }
         }
 
         protected override string RenderTestMethodBodyAssert(TestMethodBody testMethodBody)
         {
-            if (testMethodBody.CanonicalDataCase.Property == propertyEquals)
+            if (testMethodBody.CanonicalDataCase.Property == PropertyEquals)
             {
                 return base.RenderTestMethodBodyAssert(testMethodBody);
             }
-            else if (testMethodBody.CanonicalDataCase.Property != propertyToString)
+            else if (testMethodBody.CanonicalDataCase.Property != PropertyToString)
             {
                 return RenderConsistencyToAssert(testMethodBody);
             }

